assets/js/analysis/new.controller.js

```diff
@@ -231,13 +229,13 @@
             return cleanupType.unfriendly_name;
           }),
           operation: $scope.selectedAnalysis.unfriendly_name,
-          tokenizer: $scope.selectedTokenizer.unfriendly_name,
+          tokenizer: $scope.selectedTokenizer? $scope.selectedTokenizer.unfriendly_name: "",
```

Comment:
We're shifting over to make tokenizers not a necessary thing to pass to the back-end (since some libraries on the back-end manage that part for us).  But, as of now, the back-end will complain and throw errors if we don't provide something in that field; this ternary just provides an empty string to pacify the complaints for now.